src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/legaladvisor/DocAssemblyTemplateBodyMapper.java

```diff
@@ -60,6 +60,11 @@ public DocAssemblyTemplateBody from(CCDCase ccdCase, UserDetails userDetails) {
 
         LocalDate currentDate = LocalDate.now(clock.withZone(UTC_ZONE));
 
+        String respondentPartyName = ccdCase.getRespondents().get(0)
```

Comment:
Do we need to update/add a unit test that would test this condition?

Reply:
Good shout